docs/changes/2636.features.rst

```diff
@@ -0,0 +1,3 @@
+Makes changes to the CameraCalibrator in ctapipe.calib.camera.calibrator that allows it to correctly calibrate variance images generated with the VarianceExtractor
+if the VarianceExtractor is used for the CameraCalibrator the element-wise square of the relative and absolute gain calibration factors are applied to the image
```

Comment:
Please format according to ReST syntax. This will render as a single line.

Reply:
I changed it